cognite/neat/rules/models/rules.py

```diff
@@ -330,6 +310,29 @@ def is_suffix_compliant(cls, value, values):
         else:
             return value
 
+    @validator("namespace", always=True)
+    @skip_field_validator("validators_to_skip")
+    def set_namespace_if_none(cls, value, values):
+        if value is None:
+            if values["prefix"] == values["suffix"]:
+                return Namespace(f"http://purl.org/cognite/{values['prefix']}#")
+            else:
+                return Namespace(f"http://purl.org/cognite/{values['prefix']}/{values['suffix']}#")
```

Comment:
I find this slightly more readable, up to you if you agree. 
```suggestion
        if value is None:
            suffix = f"/{values['suffix']}" if values["prefix"] != values["suffix"] else ""
            return Namespace(f"http://purl.org/cognite/{values['prefix']}{suffix}#")

```